presto-docs/src/main/sphinx/sql/grant.rst

```diff
@@ -26,6 +26,10 @@ The optional ``WITH GRANT OPTION`` clause allows the grantee to grant these same
 
 For ``GRANT`` statement to succeed, the user executing it should possess the specified privileges as well as the ``GRANT OPTION`` for those privileges.
 
+.. note::
+
+    Ensure that authentication has been enabled before running any of the authorization commands.
```

Comment:
I'm confused about the purpose of this note. The SQL documentation is typically for end users. Is this note for server administrators (that the Presto server needs authentication setup) or for end users (to make sure their client is authenticating itself)?